hmsUpdateChartTables.ps1

```diff
@@ -152,11 +154,14 @@ $Query = "
 			$( DBCastDateTimeFieldAsHour 'timestamp') AS hour, 
 			COUNT(*) as numhits 
 		FROM hm_fwban_rh 
-		GROUP BY day, hour
+		GROUP BY $( DBCastDateTimeFieldAsDate 'timestamp'), $( DBCastDateTimeFieldAsHour 'timestamp')
 	) d 
 	GROUP BY hour 
 	ORDER BY hour ASC
 "
+
+Write-Host $Query
+
```

Comment:
This one runs unattended. No need for write-host. Nobody will ever see it.

Reply:
Sorry.

Reply:
Oops - I just realized that was debugging...